packages/react-select/src/Select.tsx

```diff
@@ -74,15 +71,15 @@ export interface Props<
   Group extends GroupBase<Option>
 > {
   /** HTML ID of an element containing an error message related to the input**/
-  'aria-errormessage'?: string;
+  'aria-errormessage'?: AriaAttributes['aria-errormessage'];
```

Comment:
`AriaAttributes['aria-errormessage']` includes `| undefined` so this is technically equivalent to

```
'aria-errormessage': AriaAttributes['aria-errormessage'];
```

but I'll defer to your preferences here @Methuselah96

Reply:
I'm not sure it's always equivalent ([TS playground](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAJQKYEMDG8BmUIjgcilQ3wG4Aoc4AOxiSk3STgHkwZgJqUAbABRxgAznADe5OJLh0hMAPwAuOLKg0A5nAA+cAK7UAJkkw0k+igF9KaHiiEi2HLrzhIAHnQMjk6GADoAwriQ1Ei0ADwOnNz8gkIAfGISUkQG9AAUAJSJUjlwRDA6UNRw1Do8PBQ5lpbkaFyyJVyRTjwAqkIoaswAvHAR7FHOAPRxFFS09IxozAByTQMtAhDC2VIyMEoq6lq6qcYhZuQ11rYic9TN0S7uofpexH6B4FyhMGHnl7xLwgniOSmGKCZVa5PJIApFEplCpJSTVKz1eAwAAW6gATHBeu95o4riMKEA)). Either way, I like the semantic nuance of declaring it as optional.